todoCli/todoCLI.js

```diff
@@ -0,0 +1,89 @@
+const readline = require('readline')
+const fs = require('fs')
+
+console.log("Welcome to Todo CLI!");
+console.log("----------------"); 
+
+console.log("(v) View \u2022 (n) New \u2022 (cX) Complete \u2022 (dX) Delete \u2022 (q) Quit");
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+})
+
+const todoList = []
+
+rl.on('line', data => {
+    switch (data[0]) {
+        case 'v':
+            console.log(printList())
+            console.log("(v) View \u2022 (n) New \u2022 (cX) Complete \u2022 (dX) Delete \u2022 (q) Quit");
```

Comment:
to enhance code re-usability you can create a function that returns the string of menu options and just call it here, rather than repeating console log for every case of command.
For example, `const menu = () => console.log("(v) View \u2022 (n) New \u2022 (cX) Complete \u2022 (dX) Delete \u2022 (q) Quit")`
then just do menu() in line 20, 26, 32 and 37.